src/branch_view.js

```diff
@@ -4,8 +4,8 @@ goog.require('utils');
 goog.require('banner_css');
 goog.require('safejson');
 
-function renderHtmlBlob(parent, html, bannerElement) {
-	var re, match;
+function renderHtmlBlob(parent, html, hasApp) {
+	var re, match, ctaText;
```

Comment:
Yes, because lines 19 and 25 can only be accessed if the match in line 13 exists.  If the metadata script has not been added to the template (either now or with a dashboard change in the future), the `ctaText` variable will never be set, and the CTA itself will be blank.